### DISCUSSION TITLE
Column search on Table seems to be broken

### DISCUSSION THREAD

**Discussion by rbbp23** [primefaces/community#937] [PrimeNG] [answered]
I've experienced this in my application but when I checked, it can replicate in the documentation as well.

Go to https://primeng.org/table#stateful for example, and try typing something in Search by Name field in the first column.

Result: **undefined** appears in the search box

<img width="1072" alt="Screenshot 2024-01-16 at 16 15 00" src="https://github.com/primefaces/community/assets/135729155/96c5ba36-3871-49e6-b8d5-6562a1de0ac6">

Expectation: it works and the search term shows up there, as before.

I'm not sure if this is already covered in the recent fixes planned for next milestone, if not can you please look into it?

**Comment by cetincakiroglu** [comment]
Thanks for the report we'll address the issue in the upcoming release https://github.com/primefaces/primeng/issues/14571

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

I guess it's demo related, assigning value to the input element is not necessary and that's why it become undefined. Here is the stackblitz after the removal of value from input https://stackblitz.com/edit/vdbanc
